gdbinit.py

```diff
@@ -16,8 +18,46 @@
     _start_time = time.time()
     _profiler.enable()
 
+
 # Get virtualenv's site-packages path
 venv_path = os.environ.get("PWNDBG_VENV_PATH")
+
+
+def calculate_hash(file_path):
+    return hashlib.file_digest(open(file_path, "rb"), "sha256").hexdigest()
+
+
+def run_poetry_install(dev=False):
+    command = ["poetry", "install"]
+    if dev:
+        command = ["poetry", "install", "--with", "dev"]
+    subprocess.run(command, check=True)
+
+
+PWNDBG_VENV_PATH = os.getenv("PWNDBG_VENV_PATH")
```

Comment:
Looks like the `venv_path` variable already exists, let's us that instead of making our own. I also think lower case variables are fine for all these names, as they're not constants.

Reply:
I am sorry. The duplication was unintentional. When integrating the new functionality for checking and updating dependencies, I mistakenly redefined `venv_path`, not realizing it was already declared.